### ISSUE TITLE
sequence_length parameter

### ISSUE THREAD

**Issue by fatimabs** [flairNLP/flair#1228]
what is the role of sequence_length parameter in the training process ?

**Comment by alanakbik** [comment]
Language models are trained using "truncated backpropagation through time", meaning that the RNN is unrolled only for a specific number of time steps after which gradients are detached. This parameter influences how far back the gradients travel. So the higher this value, the longer the sequence over which information is encouraged to be retained in the RNN.
